src/tests/class_tests/openms/source/QTClusterFinder_test.cpp

```diff
@@ -74,6 +74,7 @@ START_SECTION((static BaseGroupFinder* create()))
 	BaseGroupFinder* base_ptr = nullptr;
 	base_ptr = QTClusterFinder::create();
   TEST_NOT_EQUAL(base_ptr, base_nullPointer);
+  if (base_ptr != nullptr) delete base_ptr;
```

Comment:
no need for `if ...`, 
its enough to do `delete base_ptr;` since delete will check for nullptr internally.